docs/source/01_introduction/01_introduction.md

```diff
@@ -1,6 +1,6 @@
 # What is Kedro?
 
-Kedro is an open-source Python framework for creating reproducible, maintainable and modular data science code. It borrows concepts from software engineering best-practice and applies them to machine-learning code; applied concepts include modularity, separation of concerns and versioning.
+Kedro(c) is an open-source Python framework for creating reproducible, maintainable and modular data science code. It borrows concepts from software engineering best-practice and applies them to machine-learning code; applied concepts include modularity, separation of concerns and versioning.
```

Comment:
Is this intentional?

Reply:
Oh oops no this was just for testing, I thought I removed it.